help/en-US/Microsoft.PowerShell.SecretManagement.dll-Help.xml

```diff
@@ -1015,7 +1145,7 @@ The secret value must be one of five supported types:</maml:para>
       <command:parameter required="false" variableLength="true" globbing="false" pipelineInput="False" position="named" aliases="none">
         <maml:name>NoClobber</maml:name>
         <maml:Description>
-          <maml:para>When used this parameter will cause an error if the secret name already exists.</maml:para>
+          <maml:para>{{ Fill NoClobber Description }}</maml:para>
```

Comment:
I believe this is the PlatyPS template placeholder, may need to fill this description in.